### PR TITLE
Refresh into Beta

### PR DESCRIPTION
**Pull request by dpford**

### CODE REVIEW

**Comment by jimmynotjim** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
All protractor tests are passing. Lemme just check in a browser or two.

**Comment by sebworks** [comment]
:100:

**Comment by KimberlyMunoz** [comment]
I'm having issues getting this page to render in IE8 at all. 
untitled

**Comment by KimberlyMunoz** [comment]
Found the problem!
`<!--[if lt IE 9]>      <link rel="stylesheet" href="/static/css/main.ie.css"> <![endif]-->`

That stylesheet does not exist and does not get copied over to the dist folder.

**Comment by sebworks** [comment]
good catch @KimberlyMunoz

File: TEST.md [hunk 1]
```diff
@@ -100,52 +129,13 @@ Enjoy! :relieved:
 
 # Template Macro Tests
 
-## Running MacroPolo Tests
+Test the Jinja2 templates.
 
-From within the `/test/macro_tests/` directory:
+## Running MacroPolo Tests
```
Comment (anselmbradford) [inline TEST.md]: Should be `## Running Macro Polo Tests`

**Comment by jimmynotjim** [comment]
@dpford is there still a release happening this week for this PR?

**Comment by dpford** [comment]
Sure, I'll work on getting this release in now.

**Pull request merged**